usaspending_api/common/api_request_utils.py

```diff
@@ -349,6 +349,11 @@ def handle(data_set, body, serializer=None):
         except:
             raise
 
+        # If the serializer supports eager loading, set it up
+        if serializer:
+            if hasattr(serializer, "setup_eager_loading") and callable(serializer.setup_eager_loading):
+                data_set = serializer.setup_eager_loading(data_set)
+
```

Comment:
Before it's asked, this is to perform eager loading in the case that the autocomplete is returning objects, or the autocomplete field is on a nested child. The checks for `hasattr` and `callable` are safety measures if someone uses autocomplete with a serializer that does not inherit from `LimitableSerializer`